dallinger/experiment_server/experiment_server.py

```diff
@@ -1372,6 +1379,40 @@ def worker_function(event_type, assignment_id, participant_id):
     exp.log("Received an {} notification for assignment {}, participant {}"
             .format(event_type, assignment_id, participant_id), key)
 
+    if event_type == 'TrackingEvent':
+        participant = None
+        if participant_id:
+            # Lookup assignment_id to create notifications
+            participant = models.Participant.query\
+                .filter_by(id=participant_id).all()[0]
+            assignment_id = participant.assignment_id
```

Comment:
Looks like this is no longer used.

Reply:
Indeed, we don't need the assignment_id any longer (except to lookup participant if we don't have a participant_id). I'll remove this.